Shared/View/CreateActivityView.swift

```diff
@@ -61,12 +75,12 @@ struct CreateActivityView: View {
                             category: selectedCategory, // Food
                             author: "\(user.user?.id ?? "")", // author: "eOUU1RDjcphzXd0VTUDhALy6ZB53"
                             optionTitles: optionList, // Mint, Choco, Straw
-                            // FIXME: need to pass [User.ID] instead:
-                            additionalMembers: friendList
-                        ) // Might leave off friends tagging for beta?
+                            additionalMembers: friendID
+                        )
                     } catch {
-                        // TODO: handle error
-                        print(error)
+                        // TODO: handle error whenever you just print it out
+                        alertText = error.localizedDescription
+                        showingAlert = true
```

Comment:
Note that these variables are not used by anything. You'll need to actually add a `.alert` like in the other views